AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -33,6 +33,18 @@ extension BluetoothCommunicator {
 }
 
 class BluetoothManager: NSObject, BluetoothCommunicator, ObservableObject {
+    
+    class PeripheralMeasurementTimeManager {
```

Comment:
Could we move this logic to mobilePeripheralSessionManager, so that we won't add more session-related code to BluetoothManager?

Reply:
oo, I did not come up with that kind of idea - will be changed!